fenix/app/src/main/res/values-am/strings.xml

```diff
@@ -403,8 +403,6 @@
     <!-- Text for cancel button indicating that cookie banner reduction is not supported for the current site, this is shown as part of the cookie banner details view. -->
     <string name="cookie_banner_handling_details_site_is_not_supported_cancel_button">ተወው</string>
     <!-- Text for request support button indicating that cookie banner reduction is not supported for the current site, this is shown as part of the cookie banner details view. -->
-    <string name="cookie_banner_handling_details_site_is_not_supported_request_support_button" moz:RemovedIn="115" tools:ignore="UnusedResources">ድጋፍ ይጠይቁ</string>
```

Comment:
❓ Should we be removing [release - 1] version strings so 113 as the release is 114?

Also, do we need to remove from all the languages or just the values folder would suffice and the i10n automation would take care of removing from all the other locales?